openfisca_country_template/tests/situations/a_single_parent_with_a_child.yaml

```diff
@@ -0,0 +1,21 @@
+- name: "A single parent with a child situation"
+  description: A simple situation just to test
+  period: 2017-01
+  absolute_error_margin: 0
+  households:
+    - parents: ["parent1"]
+      children: ["child1"]
```

Comment:
I believe best readability practice includes internal spaces: `[ parent1 ]` is more readable than `["parent1"]` :slig